packages/http/src/lib.ts

```diff
@@ -102,7 +102,7 @@ export const $lib = createTypeSpecLibrary({
     "shared-inconsistency": {
       severity: "error",
       messages: {
-        default: "All shared routes must agree on the value of the shared parameter.",
+        default: paramMessage`Route "${"verb"} ${"path"}" is used by multiple operations with different @sharedRoute values.`,
```

Comment:
I think "@sharedRoute values" is still somewhat confusing. If I understand correctly, maybe something like:

```suggestion
        default: paramMessage`Each operation routed at "${"verb"} ${"path"}" needs to have the @sharedRoute decorator.`,
```